### PR TITLE
[no ticket][risk=no] Minor update in e2e

### PR DESCRIPTION
**Pull request by aweng98**
Code cleanup.

### CODE REVIEW

File: e2e/app/text-labels.ts [hunk 1]
```diff
@@ -1,21 +1,14 @@
 import { config } from 'resources/workbench-config';
+import { IPageUrl } from 'types';
 const { LOGIN_URL_DOMAIN_NAME, WORKSPACES_URL_PATH, ADMIN_URL_PATH, PROFILE_URL_PATH } = config;
 
-interface IPageUrl {
-  Home: string,
-  Workspaces: string,
-  Admin: string,
-  Profile: string
-}
```
Comment (aweng98) [inline e2e/app/text-labels.ts:9]: Moved to `types.d.ts`

File: e2e/app/element/radiobutton.ts [hunk 2]
```diff
@@ -31,15 +31,4 @@ export default class RadioButton extends BaseElement {
       await this.page.waitForTimeout(500);
     }
   }
-
-  /**
-   * Unselect a RadioButton.
-   */
-  async unSelect(): Promise<void> {
```
Comment (aweng98) [inline e2e/app/element/radiobutton.ts:38]: Unused

File: e2e/app/component/table.ts [hunk 3]
```diff
@@ -33,10 +18,6 @@ export default class Table extends Container {
     }
   }
 
-  async waitForVisible(): Promise<ElementHandle> {
```
Comment (aweng98) [inline e2e/app/component/table.ts:36]: Unused

File: e2e/app/component/table.ts [hunk 4]
```diff
@@ -2,17 +2,6 @@ import { ElementHandle, Page } from 'puppeteer';
 import Container from 'app/container';
 import { getPropValue } from 'utils/element-utils';
 
-// Table column names
-export enum TableColumn {
```
Comment (aweng98) [inline e2e/app/component/table.ts:6]: Unused

File: e2e/app/component/table.ts [hunk 5]
```diff
@@ -21,10 +10,6 @@ export default class Table extends Container {
     super(page, container === undefined ? xpath : `${container.getXpath()}${xpath}`);
   }
 
-  async asElement(): Promise<ElementHandle> {
-    return this.page.waitForXPath(this.xpath, { timeout: 1000, visible: true }).then((elemt) => elemt.asElement());
-  }
```
Comment (aweng98) [inline e2e/app/component/table.ts:26]: Moved to  parent class `Container.ts`

**Comment by aweng98** [comment]
@jmthibault79 FYI. This PR removes test user credentials in `.env.sample`.

**Review by jmthibault79 — APPROVED** [review]

**Pull request merged**